services/src/main/java/org/exoplatform/addons/gamification/rest/RealizationsRest.java

```diff
@@ -82,6 +84,9 @@ public Response getAllRealizations(@Parameter(description = "result fromDate", r
                                                 @QueryParam("sortDescending")
                                                 @DefaultValue("true")
                                                 boolean sortDescending,
+                                                @Parameter(description = "Identifier of the connected user", required = true)
+                                                @QueryParam("earnerId")
+                                                Long earnerId,
```

Comment:
```suggestion
                                                long earnerId,
```
To avoid having it `null`, but `0` as default value

Reply:
done